test/basic-test.html

```diff
@@ -273,6 +273,20 @@
         expect(keys.keyboardEventMatchesKeys(event, 'up')).to.be.equal(true);
       });
     });
+
+    suite('matching keyboard events to top row and number pad digit keys', function() {
+      test('top row can be done imperatively', function() {
+        var event = new CustomEvent('keydown');
+        event.keyCode = 49;
+        expect(keys.keyboardEventMatchesKeys(event, '1')).to.be.equal(true);
+      });
+
+      test('number pad digits can be done imperatively', function() {
+        var event = new CustomEvent('keydown');
+        event.keyCode = 97;
+        expect(keys.keyboardEventMatchesKeys(event, '1')).to.be.equal(true);
```

Comment:
We set the `keyCode` only in these 2 tests, in the others we set `key`.
Also, the new function would rely on the existence of `keys` and hide a check :p